src/fsharp/SyntaxTree.fsi

```diff
@@ -1047,6 +1050,12 @@ type SynExpr =
         synStringKind :SynStringKind *
         range: range
 
+    /// Debug points arising from computation expressions
+    | DebugPoint of
```

Comment:
@dsyme Since these expressions are only created during type check stage, could it be better to produce typed tree nodes instead?

Reply:
They are produced by desugaring computation expressions, which is done as SyntaxTree -> SyntaxTree. It's not easy to internalise the individual nodes produced